### PR TITLE
Lots, mostly website changes

### PR DESCRIPTION
**Pull request by swz-git**
screenshot

### CODE REVIEW

File: website/index.html [hunk 1]
```diff
@@ -72,11 +79,25 @@ <h2>Output</h2>
...
+        <select name="languages" id="language-selector">
+          <option value="julia">Julia</option>
+          <option value="ruby">Ruby</option>
+        </select>
+        <script>
+          const updateLang = () => {
+            monaco.editor.setModelLanguage(window.editor.getModel(), document.getElementById("language-selector").value)
+          }
+          document.getElementById("language-selector").onchange = updateLang
+          updateLang()
```
Comment (Andriamanitra) [inline website/index.html:95]: `monaco.editor` is not yet defined when this runs for the first time, this should probably happen when page is fully loaded

File: website/index.html [hunk 2]
```diff
@@ -33,31 +33,38 @@ <h2>Test cases</h2>
       <div>
           <h2>Output</h2>
           <div id="output" class="mono-box">
-              <p id="stderr"></p>
-              <p id="stdout"></p>
+            <pre>
+              <div id="stderr">Error: Testing</div>
+              <div id="stdout">Hello, world!</div>
+            </pre>
```
Comment (Andriamanitra) [inline website/index.html:39]: Wrapping this whole thing in `<pre>` causes some unwanted newlines and whitespace, changing the inner `div`s to `pre` is probably what we want instead?

**Review by Andriamanitra — COMMENTED** [review]
Good job, it looks great!

File: website/index.html (hunk 2 above)
Comment (swz-gh) [reply website/index.html:39]: Whoops, that was supposed to be a div with `<pre>`s inside

**Pull request merged**